tools/RAiDER/delayFcns.py

```diff
@@ -167,7 +155,64 @@ def make_interpolator(xs, ys, zs, data):
     )
 
 
-def process_chunk(k, chunkInds, SP, SLV, in_shape, stepSize, ifWet, ifHydro, max_len, wm_file):
+def chunk(chunkSize, in_shape):
```

Comment:
These chunk creation functions are all great candidates for adding unit tests. This will also help document what the function actually does since you will have something like:

```python3
assert chunk(10, (100, )) == [10, 10, 10, 10, 10, 10, 10, 10, 10, 10]
```
or whatever it actually does. I have no idea what the output is supposed to be from just reading the docstring.